apps/accounts/automations/change_secret/manager.py

```diff
@@ -27,17 +26,6 @@ class ChangeSecretManager(BaseChangeSecretPushManager):
     def method_type(cls):
         return AutomationTypes.change_secret
 
-    def get_secret(self, account):
-        if self.secret_strategy == SecretStrategy.custom:
-            new_secret = self.execution.snapshot['secret']
-        else:
-            generator = SecretGenerator(
-                self.secret_strategy, self.secret_type,
-                self.execution.snapshot.get('password_rules')
-            )
-            new_secret = generator.get_secret()
-        return new_secret
-
     def gen_account_inventory(self, account, asset, h, path_dir):
         record = self.get_or_create_record(asset, account, h['name'])
         new_secret, private_key_path = self.handle_ssh_secret(account.secret_type, record.new_secret, path_dir)
```

Comment:
No significant differences were found between the two codes provided. Therefore, there is no need for further analysis.